src/components/CTABlock.tsx

```diff
@@ -31,19 +32,37 @@ export const CTABlock = ({ content, prompt, buttonText, fireEvent, waitForEvent
 
   const maybeButton = (): JSX.Element => {
     if (!clicked) {
-      return (<>
-      <button onClick={clickHandler} type="button" className="btn btn-outline-primary mb-3">{buttonText}</button></>)
+      return (
+        <>
+          <button
+            onClick={clickHandler}
+            type="button"
+            className={ctaStyleTheme === "green" ? "os-submit-button-style-theme" : "os-submit-button-default-theme"}>
```

Comment:
I think the trailing `>` here can be removed.

Reply:
I think we need to keep the trailing `>` because it closes `<button`.

Reply:
Ah, right. My :eyes:  missed that.